sharktank/sharktank/ops/default_impls.py

```diff
@@ -602,7 +602,7 @@ def to_default(tensor: Tensor, *args, **kwargs) -> Tensor:
 
 
 @trace_tensor.override(AllOfExprsVariadic(IsOfType(Tensor, InferenceTensor)))
-def trace_tensor(key: str, *tensors: tuple[AnyTensor]):
+def trace_tensor(key: str, *tensors: tuple[Union[Tensor, InferenceTensor], ...]):
```

Comment:
`AnyTensor` is defined as `Union[Tensor, InferenceTensor]`

Reply:
You are right. Fixed it.